resolution: 0.2
map_length: 30
sensor_noise_factor: 0.02
mahalanobis_thresh: 2.0
outlier_variance: 0.01
drift_compensation_variance_inler: 0.1
time_variance: 0.01
max_variance: 100.0
initial_variance: 100.0
traversability_inlier: 0.1
dilation_size: 2
wall_num_thresh: 100
min_height_drift_cnt: 100
position_noise_thresh: 0.001
orientation_noise_thresh: 0.001
position_lowpass_alpha: 0.1
orientation_lowpass_alpha: 0.1
min_valid_distance: 0.3
max_height_range: 1.0
recordable_fps: 3.0

max_ray_length: 20.0
cleanup_step: 0.01

gather_mode: 'mean'
enable_edge_sharpen: true
enable_visibility_cleanup: true
enable_drift_compensation: true
enable_pointcloud_publishing: false

weight_file: 'config/weights.yaml'

pointcloud_topics: ['/point_cloud_filter/velodyne/point_cloud_cropped']
# pointcloud_topics: ['/velodyne/point_cloud']
pose_topic: '/state_estimator/pose_in_odom'
# pose_topic: '/state_estimator_standalone/pose_in_odom'
map_frame: 'odom'
